frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import Button from './Button'
+import styled from 'styled-components'
+import { useHistory } from 'react-router-dom'
+import { PATHS } from '../routes/Routes'
+import { JsxElement } from 'typescript'
```

Comment:
is this used?

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.